Lib/idlelib/autocomplete_w.py

```diff
@@ -256,7 +256,7 @@ def winconfig_event(self, event):
         else:
             # place acw above current line
             new_y -= acw_height
-        acw.wm_geometry("+%d+%d" % (new_x, new_y))
+        self.widget.after(1, lambda: self._update_geometry(acw, new_x, new_y))
```

Comment:
The call could be delayed without adding two layers of wrapping.
```suggestion
        self.widget.after(1, acw.wm_geometry, "+%d+%d" % (x, y)) 
```
and delete the added _update_geometry.  But I don't think this is the right fix.

Reply:
Personally I think the lambda approach is more readable, but fair enough.